smdebug/tensorflow/base_hook.py

```diff
@@ -441,7 +445,10 @@ def set_optimizer_variables(self, optimizer_variables):
         # TF 2.x doesn't provide gradient/optimizer variable names and values by default.
         # Skipping set_gradients and set_optimizer_variables for Tf 2.x until there is
         # support to pass names and values from TF side.
-        if is_tf_version_2x() and tf.executing_eagerly():
+
+        # From TF 2.2, executing_eagerly_outside_functions() should be called using
+        # tf.compat.v1.executing_eagerly_outside_functions()
```

Comment:
Likewise

Reply:
this release will go into TF 2.1 container which wont have this name. so we'll have to make the change for the next container. it fails now if i use tf.compat.v1